Enterprise-CMCS/macpro-onemac
# Fix overflow for actions menu

Story: https://qmacbis.atlassian.net/browse/OY2-23749

### Details

Removes unnecessary (and breaking) overflow-x control in style class to fix the bug.

### Changes

- Change `overflow-x` control for `dashboard-and-alert-bar`:  was `auto`, now `scroll`. Just a more declarative way of styling is all. Functionality should remain the same
- Turn `#root` from a flex-box column to a flex-box column _with `space-between` justification_. This makes the headers and footers stick to the top and bottom of a viewport while the middle can be its own content container.
- Add `padding-bottom` to the table's container to allow for y-axis pop-up menu space without scrolling. Because of its need to be `absolute` in its positioning `relative` to its parent, we cannot rely on `overflow-y` controls.

### Implementation Notes

#### Before
<img width="400" alt="Screen Shot 2023-05-02 at 3 13 51 PM" src="https://user-images.githubusercontent.com/4022304/235763362-88730617-acd0-4e4b-8a53-93f3d26f7d21.png">
<img width="400" alt="Screen Shot 2023-05-02 at 3 14 04 PM" src="https://user-images.githubusercontent.com/4022304/235763368-3274aa7e-497f-4b12-945e-da2059fb866a.png">


#### After

<img width="400" alt="Screen Shot 2023-05-02 at 3 08 34 PM" src="https://user-images.githubusercontent.com/4022304/235763143-c04bb45b-2da6-4736-911f-1b6192254586.png">
<img width="400" alt="Screen Shot 2023-05-02 at 3 09 05 PM" src="https://user-images.githubusercontent.com/4022304/235763147-fe687fdc-7fc9-4d2a-a8e5-b804209a6002.png">


### Test Plan

1. Grab any ID on a Dashboard row that has package actions available in the 3-dot menu
2. Enter the exact ID in search to ensure it's the only item on the table
3. Click the 3-dot menu under Actions column
4. Ensure the fixed styles are represented ("After" above)
